api/v3/utils.php

```diff
@@ -1077,6 +1077,11 @@ function _civicrm_api3_object_to_array_unique_fields(&$dao, &$values) {
  *   ID of entity per $extends.
  */
 function _civicrm_api3_custom_format_params($params, &$values, $extends, $entityId = NULL) {
+  if (!empty($params['custom'])) {
```

Comment:
@seamuslee001 just thinking about this - is there any risk of bypassing validation? If so I think we could put in a temporary fix to prevent it being accessed outside the php layter until we have removed it - ie set a static before the call that can only be set in the php layer so this couldn't be accessed via json

Reply:
I don't think so I suppose we can put in some extra validation if you are concerned

Reply:
yeah - I do hope this is a temporary pass-through but I'd hate to get the wrong credits next security release